pysmurf/tune/smurf_tune.py

```diff
@@ -2470,52 +2264,84 @@ def track_and_check(self, band, channel=None, reset_rate_khz=4.,
         relock=True, tracking_setup=True,
         feedback_start_frac=None, feedback_end_frac=None):
         """
-        This runs tracking setup and check_lock to prune bad channels.
+        This runs tracking setup and check_lock to prune bad channels. This has
```

Comment:
@eyyoung24 could you please also fix the lms_gain hardcode here, and add to the reset_rate_khz hardcode issue (or fix).